modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/util/PowwowUtil.java

```diff
@@ -473,6 +386,27 @@ public static void sendNotifications(
 		powwowSubscriptionSender.flushNotificationsAsync();
 	}
 
+	public static void sendNotifications(
```

Comment:
should we name it in a way that can tell other developers know why the logic inside this method is different to the other  `sendNotifications`?